hipcub/PKGBUILD

```diff
@@ -2,46 +2,29 @@
 pkgname=hipcub
 pkgver=3.3.0
 pkgrel=2
-pkgdesc="Reusable software components for the rocm developers."
+pkgdesc='header-only library on top of rocPRIM or CUB'
 arch=('x86_64')
 url="https://github.com/ROCmSoftwarePlatform/hipCUB"
-license=('custom:NCSAOSL')
-depends=('hcc' 'hip')
-makedepends=('cmake' 'python2' 'rocminfo' 'comgr')
-source=("https://github.com/ROCmSoftwarePlatform/hipCUB/archive/rocm-$pkgver.tar.gz")
+license=('custom')
+depends=('rocprim')
+optdepends=('cuda: CUB backend')
+makedepends=('cmake' 'git' 'hcc')
+source=("$url/archive/rocm-$pkgver.tar.gz")
 sha256sums=('0d97cada3b856e8c62affbb3664ac4fc2415a36ad3d9b9d7155b5f954e492392')
 
 build() {
-  mkdir -p "$srcdir/build"
-  cd "$srcdir/build"
-
-  # Tensile library needs python to be python2
-  export PATH="$srcdir:$PATH"
-  [[ -e "$srcdir/python" ]] || ln -s /usr/bin/python2 "$srcdir/python"
-
-  # fix broken build with stack protection
-  export CFLAGS="$(sed -e 's/-fstack-protector-strong//' <<< "$CFLAGS")"
-  export CXXFLAGS="$(sed -e 's/-fstack-protector-strong//' <<< "$CXXFLAGS")"
-  export CPPFLAGS="$(sed -e 's/-fstack-protector-strong//' <<< "$CPPFLAGS")"
+  mkdir -p build
+  cd build
 
-  # compile with HCC
-  export CXX="/opt/rocm/hcc/bin/hcc"
-
-  # TODO: fix libhipcub.so, it contains references to $srcdir
-  cmake -DCMAKE_BUILD_TYPE=Release \
-        -DCMAKE_INSTALL_PREFIX=/opt/rocm/hipcub \
+  CXX=/opt/rocm/hcc/bin/hcc \
+  cmake -DCMAKE_INSTALL_PREFIX=/opt/rocm/hipcub \
         -Damd_comgr_DIR=/opt/rocm/lib/cmake/amd_comgr \
         "$srcdir/hipCUB-rocm-$pkgver"
-  make
```

Comment:
don't we need make here?

Reply:
hipcub is a header-only library. If you don't want to run the tests / benchmarks, you don't need to call make.

Reply:
kk I see, TIL